edge_bench_3/edge_ren_1.rs

```diff
@@ -0,0 +1,44 @@
+
+//! Autogenerated weights for edge_ren
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 2.0.0
+//! DATE: 2021-01-05, STEPS: [2, ], REPEAT: 2, LOW RANGE: [], HIGH RANGE: []
+//! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("dev"), DB CACHE: 128
+
+// Executed Command:
+// ./target/release/edgeware
+// benchmark
+// --chain
+// dev
+// --execution=wasm
+// --wasm-execution=compiled
+// --pallet
+// edge_ren
+// --extrinsic
+// validate_and_mint
+// --steps
+// 2
+// --repeat
+// 2
+// --output
+// ./edge_bench_3/edge_ren_1.rs
+// --raw
+
+
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+
+use frame_support::{traits::Get, weights::Weight};
+use sp_std::marker::PhantomData;
+
+/// Weight functions for edge_ren.
+pub struct WeightInfo<T>(PhantomData<T>);
+impl<T: frame_system::Config> edge_ren::WeightInfo for WeightInfo<T> {
+	fn validate_and_mint(z: u32, ) -> Weight {
```

Comment:
seems there are 3 implementations of weighs, which are all different. This seems really unncessary and doesn' provide any clarity as to which bench is worthy. Please use one consistent flow.

Reply:
The edge_bench_* folders and the files containing them are only for you to see the results of the benchmarks of validate_and_mint with various ranges of "z". This information would be helpful in giving context to my benching concern. They were never intended to be part of the commit that would be merged.
The weights.rs file actually in use is in the edge-ren folder, as it should be.

Reply:
Ok I would move these out nonetheless.